public/app/core/components/PageHeader/PageHeader.tsx

```diff
@@ -47,8 +55,14 @@ const SelectNav = ({ main, customCss }: { main: NavModelItem; customCss: string
 };
 
 const Navigation = ({ main }: { main: NavModelItem }) => {
+  const { children } = main;
+
+  if (!children?.length) {
```

Comment:
:exploding_head:  this is a confusing way to write if(!children || children.length === 0)   but might just be me who not a big fan of the ? operator

Reply:
@torkelo sure, so can change it to the longer version if you prefer.